### ISSUE TITLE
ISO Schematron validation does not support include mechanism

### ISSUE THREAD

**Issue by ujay68** [relaxng/jing-trang#258]
If you look at https://github.com/relaxng/jing-trang/blob/master/mod/schematron/src/main/com/thaiopensource/validate/schematron/resources/iso-schematron.rnc, the ISO Schematron dialect implemented by Jing seems to be one supporting the include mechanism. However, using an `include` element in a schema doesn't do anything. The implementing XSLT script https://github.com/relaxng/jing-trang/blob/master/mod/schematron/src/main/com/thaiopensource/validate/schematron/resources/iso-schematron.xsl doesn't process`include` elements in its input file.

**Comment by georgebina** [comment]
ISO Schematron is not really supported by Jing, see https://github.com/relaxng/jing-trang/issues/23.

**Comment by ujay68** [comment]
@georgebina, thanks for the pointer. Would you suggest to use the old "http://www.ascc.net/xml/schematron" schematron version then? Seems to be functionally equivalent as long as you don't (and cannot) use more than the basic features (pattern/rules/assert) anyway.

**Comment by georgebina** [comment]
Yes, the pre-ISO schematron is supported by Jing and you should be able to use that.
